src/qt/guiutil.cpp

```diff
@@ -597,20 +600,33 @@ bool SetStartOnSystemStartup(bool fAutoStart)
             TCHAR pszExePath[MAX_PATH];
             GetModuleFileName(NULL, pszExePath, sizeof(pszExePath));
 
-            TCHAR pszArgs[5] = TEXT("-min");
+            // Start client minimized
+            QString strArgs = "-min";
+            // Start testnet client, if we are currently running testnet
+            if (Params().NetworkIDString() == "test")
```

Comment:
If you do keep it like this based on special-casing the network name, make sure that you avoid doing anything if the net is not `main` or `test`